LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 Thomas Waldmann
+Copyright (c) 2018 Thomas Waldmann (see AUTHORS file)
+Copyright (c) 2021 Wilko Nienhaus (see AUTHORS file)
```

Comment:
Maybe it is simpler to update authors only at one place, so I suggest this:

Copyright 2018-2021 by the py-esp32-ulp authors, see AUTHORS file.

Also, I would not go into details about who has written what, that's where git log is better anyway and a file describing this will likely be always outdated.

Reply:
Done. 47d5e8a